File: examples/inspect_mobile_safari/expturlhandler/ViewController.m
//
//  ViewController.m
//  expturlhandler
//
//  Created by Faisal Memon on 07/11/2021.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    [[UIApplication sharedApplication] openURL:[NSURL URLWithString:@"x-web-search://?toast"]
                                       options:@{}  completionHandler:^(BOOL success) {
        NSLog(@"status %ld", (long)success);
    }];
}


@end

File: source/BookBuilder/BookBuilder/ViewController.swift
//
//  ViewController.swift
//  BookBuilder
//
//  Created by Faisal Memon on 14/05/2021.
//

import Cocoa
import BuilderLibrary


class ViewController: NSViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let config = Configuration.en
        BuilderLibrary.buildWithConfiguration(config)
        
        // Do any additional setup after loading the view.
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

